net.meshlab.MeshLab.yaml

```diff
@@ -0,0 +1,78 @@
+app-id: net.meshlab.MeshLab
+runtime: org.kde.Platform
+sdk: org.kde.Sdk
+runtime-version: '5.15'
+command: meshlab
+rename-desktop-file: meshlab.desktop
+# Wayland does not work https://github.com/cnr-isti-vclab/meshlab/issues/738
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+cleanup:
+  - /include
+  - /lib/cmake
+  - /lib/pkgconfig
+  - /share/doc
+  - /share/man
+  - /bin/rbox
+  - /bin/q*
+  - /bin/3dsdump
+  - "*.la"
+  - "*.a"
+
+modules:
+  - shared-modules/glew/glew.json
+  - shared-modules/glu/glu-9.json
+  
+  - name: Eigen3
+    buildsystem: cmake-ninja
+    builddir: true
+    cleanup: 
+      - "*"
+    sources:
+      - type: archive
+        url: https://gitlab.com/libeigen/eigen/-/archive/3.3.9/eigen-3.3.9.tar.bz2
+        sha256: 0fa5cafe78f66d2b501b43016858070d52ba47bd9b1016b0165a7b8e04675677
+        
+  - name: lib3ds
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/lib3ds/lib3ds-1.3.0.zip
+        sha256: f5b00c302955a67fa5fb1f2d3f2583767cdc61fdbc6fd843c0c7c9d95c5629e3
+      - type: script
+        dest-filename: autogen.sh
+        commands:
+          - autoreconf -vfi
+        
+  - name: gmp
+    config-opts:
+      - --prefix=/app
+    sources:
+      - type: archive
+        url: https://ftp.gnu.org/gnu/gmp/gmp-6.2.1.tar.xz
+        sha256: fd4829912cddd12f84181c3451cc752be224643e87fac497b69edddadc49b4f2
+  
+  - name: MeshLab
```

Comment:
But sadly MeshLab fails to build on `aarch64` because of GL.
```
[ 12%] Building CXX object external/u3d/CMakeFiles/IFXCoreStatic.dir/src/RTL/Kernel/DataTypes/IFXCoreList.cpp.o
In file included from /run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/../../ssynth/SyntopiaCore/GLEngine/EngineWidget.h:12,
                 from /run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/moc_EngineWidget.cpp:10,
                 from /run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/mocs_compilation.cpp:2:
/run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/../../ssynth/SyntopiaCore/GLEngine/Object3D.h: In member function ‘void SyntopiaCore::GLEngine::Object3D::vertex(SyntopiaCore::Math::Vector3f) const’:
/run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/../../ssynth/SyntopiaCore/GLEngine/Object3D.h:70:56: error: ‘glVertex3f’ was not declared in this scope; did you mean ‘vertex3n’?
   70 |    void vertex(SyntopiaCore::Math::Vector3f v) const { glVertex3f(v.x(), v.y(), v.z()); }
      |                                                        ^~~~~~~~~~
      |                                                        vertex3n
/run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/../../ssynth/SyntopiaCore/GLEngine/Object3D.h: In member function ‘void SyntopiaCore::GLEngine::Object3D::normal(SyntopiaCore::Math::Vector3f) const’:
/run/build/MeshLab/src/external/structuresynth-1.5/external-ssynth_autogen/TWFB33PXDR/../../ssynth/SyntopiaCore/GLEngine/Object3D.h:71:56: error: ‘glNormal3f’ was not declared in this scope
   71 |    void normal(SyntopiaCore::Math::Vector3f v) const { glNormal3f(v.x(), v.y(), v.z()); }
      |                                                        ^~~~~~~~~~
make[2]: *** [external/structuresynth-1.5/CMakeFiles/external-ssynth.dir/build.make:82: external/structuresynth-1.5/CMakeFiles/external-ssynth.dir/external-ssynth_autogen/mocs_compilation.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2008: external/structuresynth-1.5/CMakeFiles/external-ssynth.dir/all] Error 2
```

Reply:
Yeah, I guess aarch64 should be disabled.